language-plutus-core/generators/Language/PlutusCore/Generators/Internal/TypedBuiltinGen.hs

```diff
@@ -98,34 +89,24 @@ updateTypedBuiltinGen tbNew genX genTb tbOld
 updateTypedBuiltinGenSized
     :: (Monad m, PrettyDynamic a)
     => TypedBuiltinSized a  -- ^ A generator of which sized built-in to overwrite.
-    -> (Size -> GenT m a)   -- ^ A function that computes new generator from a 'Size'.
+    -> GenT m a             -- ^ A new generator
```

Comment:
It can be a problem here. Generators do not receive sizes only because we have sizes in the type system, but also because we literally want to control the sizes of integers and bytestrings that we generate, because otherwise we get constants that are simply too big. And even though this was a very carefully crafted system, it still generated out of bounds errors in like half the cases. Now that we do not control sizes, we can easily generate expressions that evaluate to something way too big. So I guess we should at the very least hardcode smaller sizes for now, see below.